crates/bevy_ecs/src/label.rs

```diff
@@ -150,6 +150,12 @@ macro_rules! define_label {
             }
         }
 
+        impl ::core::borrow::Borrow<dyn $label_trait_name> for $crate::intern::Interned<dyn $label_trait_name> {
```

Comment:
Should this be a blanket impl on `Interned`?  It seems like it would be valuable even for things that aren't labels, and generics are a little simpler than macros.

Reply:
Oh, wait, no, `Borrow` is only valid if the `Eq` and `Hash` impls match, which won't be true in general.